README.md

```diff
@@ -88,7 +88,7 @@ docker-compose --env-file compose.env up --build
 -   The compose file uses `compose.env` file for environment variables. You can modify it as per your needs.
 -   The compose file will start the following services:
     -   `ollama-server` - Used to start local LLM server. `compose.env` has `OLLAMA_MODEL` as the environment variable to specify the model.
-    -   `infinity-server` - Used to start local embeddings and rerankers server. `compose.env` has `INFINITY_EMBEDDING_MODEL` and `INFINITY_RERANKING_MODEL` as the environment variable to specify the embedding and reranker.
+    -   `infinity-server` - Used to start local embeddings and rerankers server. `compose.env` has `INFINITY_MODEL_ID` as the environment variable to specify the embedding and reranker.
```

Comment:
Let's revert this and instead format the `INFINITY_MODEL_ID` in the docker compose itself
Easier for the users to manage them differently

Reply:
Are you sure? Feel free to commit on the branch!